### ISSUE TITLE
compute/lecture/notifications: a subtle mention on the lack of queuing of notifications

### ISSUE THREAD

**Issue by mateibarbu19** [open-education-hub/operating-systems#150]
In the [lecture](https://github.com/open-education-hub/operating-systems/blame/master/content/chapters/compute/lecture/slides/notifications.md#L72), the listings of the `notify` API on condition variables lacks to mention that if a notification is sent while no thread is waiting on that variable, that notification is lost.

This is appears to be the case both in [Python 3.11](https://github.com/python/cpython/blob/b7a49eb5ff541157a87fefb8fa0527a5cf2b3abc/Lib/threading.py#L371) and Pthreads' cond type

> The pthread_cond_broadcast() and pthread_cond_signal() functions shall have no effect if there are no threads currently blocked on cond. - `man pthread_cond_broadcast`

This seems worth mentioning to me.

What do you think?

**Comment by teodutu** [comment]
Yes, this would be a nice thing to mention I think. It's definitely not a piece of critical info, but it will maybe help students prevent some potential deadlocks. @cristian-vijelie go for it!
